scripts/install/install-linux-arm64.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+# Installs the porter CLI for a single user.
+# PORTER_HOME:      Location where Porter is installed (defaults to ~/.porter).
+# PORTER_MIRROR:       Base URL where Porter assets, such as binaries and atom feeds, are downloaded. This lets you
+#                   setup an internal mirror.
+# PORTER_PERMALINK: The version of Porter to install, such as vX.Y.Z, latest or canary.
+# PKG_PERMALINK:    The version of mixins and plugins to install, such as latest or canary.
+
+export PORTER_HOME=${PORTER_HOME:-~/.porter}
+export PORTER_MIRROR=${PORTER_MIRROR:-https://cdn.porter.sh}
+PORTER_PERMALINK=${PORTER_PERMALINK:-latest}
+PKG_PERMALINK=${PKG_PERMALINK:-latest}
+
+echo "Installing porter@$PORTER_PERMALINK to $PORTER_HOME from $PORTER_MIRROR"
+
+mkdir -p $PORTER_HOME/runtimes
+
+curl -fsSLo $PORTER_HOME/porter $PORTER_MIRROR/$PORTER_PERMALINK/porter-linux-arm64
+chmod +x $PORTER_HOME/porter
+cp $PORTER_HOME/porter $PORTER_HOME/runtimes/porter-runtime
```

Comment:
Have you tested out using porter-linux-arm64 installed to ~/.porter/runtimes/porter-runtime? Like built a bundle and run it?

I'm not sure that will work without further changes because we build bundles with a linux amd64 invocation image.

Reply:
sorry for the delay to reply. No, but I will test, just need to setup a linux-arm64  machine, thanks for the review and feedback!

Reply:
Just checking if you tried this out yet and how it went?

Reply:
@carolynvs sorry :( i was busy with company work and did not have time, I will take some days off and when I comeback i will get this back on track. I will be out starting today till next Tuesday and on Wednesday I will run all the tests on this.

Reply:
No need to apologies! I just ping issues after it's been a while to see how things are going and make sure that people aren't stuck. Take your time and I appreciate you looking into this. 👍

Reply:
I will publish the packages in a dummy github repo to be able to test, but this is what I got so far in a Linux arm64 machine

```
ubuntu@ip-172-31-10-226:~/porter/porter$ ./scripts/install/install-linux-arm64.sh
+ export PORTER_HOME=/home/ubuntu/.porter
+ PORTER_HOME=/home/ubuntu/.porter
+ export PORTER_MIRROR=https://cdn.porter.sh
+ PORTER_MIRROR=https://cdn.porter.sh
+ PORTER_PERMALINK=latest
+ PKG_PERMALINK=latest
+ echo 'Installing porter@latest to /home/ubuntu/.porter from https://cdn.porter.sh'
Installing porter@latest to /home/ubuntu/.porter from https://cdn.porter.sh
+ mkdir -p /home/ubuntu/.porter/runtimes
+ cp /home/ubuntu/porter/porter/bin/v0.38.3-11-g6f9ae84e/porter-linux-arm64 /home/ubuntu/.porter/porter
+ chmod +x /home/ubuntu/.porter/porter
+ cp /home/ubuntu/.porter/porter /home/ubuntu/.porter/runtimes/porter-runtime
++ /home/ubuntu/.porter/porter version
+ echo Installed porter v0.38.3-11-g6f9ae84e '(6f9ae84e)'
Installed porter v0.38.3-11-g6f9ae84e (6f9ae84e)
+ /home/ubuntu/.porter/porter mixin install exec --version latest
Error: exec @ latest did not publish a download for linux/arm64
```

Reply:
published the `exec` binary but looks like it tried to get the linux/amd64

```
ubuntu@ip-172-31-10-226:~/porter/porter$ porter mixin install exec --url https://github.com/cpanato/testing-ci-providers/releases/download/  --version v0.0.51
Error: bad status returned when downloading https://github.com/cpanato/testing-ci-providers/releases/download/v0.0.51/exec-linux-amd64 (404) 404 Not Found
```

Reply:
made the change and then when I run it again apparently it works
```
ubuntu@ip-172-31-10-226:~/porter/porter$ porter mixin install exec --url https://github.com/cpanato/testing-ci-providers/releases/download/  --version v0.0.51
installed exec mixin v0.38.3-11-g6f9ae84e (6f9ae84e)
```

Reply:
ahh! we will need to build the other mixins for the others like kubernetes/helm ... I get it now

sorry doing some rubber duck here :)

Reply:
Yes that would be great! 💯

Reply:
ok, will work on those things 😃